components/navigation.css

```diff
@@ -76,7 +75,7 @@ header {
     flex-direction: column;
```

Comment:
use helper class for flex so you don't have to define repeat it

Reply:
Sure. Will add more helper classes. Thanks!

Reply:
your welcome Apurva.